### PR TITLE
Cleanup packaging to make it more suitable for the release job

### PR DESCRIPTION
**Pull request by jiria**

### CODE REVIEW

**Comment by jiria** [comment]
Last minute testing found another issue with the release cab. Working on a fix, will include it here.

File: builds/misc/packages.yaml [hunk 1]
```diff
@@ -233,8 +223,20 @@ jobs:
         inputs:
           workingDirectory: 'edgelet/hsm-sys/azure-iot-hsm-c/build'
           cmakeArgs: '--build . --config Release'
-      - powershell: edgelet/build/windows/package.ps1
-        displayName: Package iotedged, iotedge, libiothsm and docker/dockerd
+      - powershell: edgelet/build/windows/package.ps1 -CreateTemplate
+        displayName: Prepare package template
+      # - task: SFP.build-tasks.custom-build-task-1.EsrpCodeSigning@1
+      #   displayName: 'ESRP Package Catalog CodeSigning Internal'
+      #   inputs:
```
Comment (jiria) [inline builds/misc/packages.yaml]: Disabling until we have the right certificate.

File: scripts/windows/setup/IotEdgeSecurityDaemon.ps1 [hunk 2]
```diff
@@ -1497,16 +1498,24 @@ function Remove-IotEdgeContainers {
...
+    $prefix = ""
+    # in case the installation has not been completed
+    if (-not (Get-Command "docker.exe" -ErrorAction SilentlyContinue)) {
+        $prefix = "$MobyInstallDirectory\"
+    }
+
     switch ($ContainerOs) {
         'Linux' {
-            return '"docker"'
+            return ('"{0}docker"' -f $prefix)
```
Comment (arsing) [inline scripts/windows/setup/IotEdgeSecurityDaemon.ps1]: Shouldn't do this for Linux. We want to use Docker for Windows' `docker.exe` in that case.

Comment (jiria) [reply scripts/windows/setup/IotEdgeSecurityDaemon.ps1]: Fixed.

**Pull request merged**